main.go

```diff
@@ -103,6 +108,10 @@ func main() {
 						`))
 	})
 
+	if isOnCloudFoundry() {
```

Comment:
Why do this explicitly and not via the cmd line config?

Reply:
See above.